src/components/button/button.ts

```diff
@@ -70,6 +70,11 @@ export enum BUTTON_SIZE {
    */
   SMALL = 'sm',
 
+  /**
+   * X-Large size.
+   */
+  XLARGE = 'xl',
```

Comment:
To align to: https://github.com/carbon-design-system/carbon-web-components/blob/v1.8.0/src/components/dropdown/dropdown.ts#L82and https://github.com/carbon-design-system/carbon-web-components/blob/v1.8.0/src/components/input/input.ts#L46

```suggestion
  EXTRA_LARGE = 'xl',
```

Reply:
Thanks @asudoh!